app/shared/documents/canvas.tsx

```diff
@@ -6,35 +6,40 @@ import {
   Scripts,
   ScrollRestoration,
 } from "@remix-run/react";
+import { ErrorBoundary, withSentryRouteTracing } from "@sentry/remix";
 import { CriticalCss } from "@webstudio-is/sdk";
 import { Env } from "~/shared/env";
 
 /**
  * We are using Outlet prop from index layout when user renders site from a subdomain.
  */
-export const Canvas = ({
+const CanvasRoot = ({
   Outlet = DefaultOutlet,
 }: {
   Outlet: typeof DefaultOutlet;
 }) => {
   return (
-    <html lang="en">
-      <head>
-        <meta charSet="utf-8" />
-        <meta name="viewport" content="width=device-width,initial-scale=1" />
-        <link rel="icon" href="/favicon.ico" type="image/x-icon" />
-        <link rel="shortcut icon" href="/favicon.ico" type="image/x-icon" />
-        <Meta />
-        <Links />
-        <CriticalCss />
-      </head>
-      <body>
-        <Outlet />
-        <ScrollRestoration />
-        <Env />
-        <Scripts />
-        {process.env.NODE_ENV === "development" && <LiveReload />}
-      </body>
-    </html>
+    <ErrorBoundary>
+      <html lang="en">
+        <head>
+          <meta charSet="utf-8" />
+          <meta name="viewport" content="width=device-width,initial-scale=1" />
+          <link rel="icon" href="/favicon.ico" type="image/x-icon" />
+          <link rel="shortcut icon" href="/favicon.ico" type="image/x-icon" />
+          <Meta />
+          <Links />
+          <CriticalCss />
+        </head>
+        <body>
+          <Outlet />
+          <ScrollRestoration />
+          <Env />
+          <Scripts />
+          {process.env.NODE_ENV === "development" && <LiveReload />}
+        </body>
+      </html>
+    </ErrorBoundary>
   );
 };
+
+export const Canvas = withSentryRouteTracing(CanvasRoot);
```

Comment:
Error happens here, seems to be `@sentry/remix` types not allowing for props but app seems to work the same

Reply:
withSentryRouteTracing has a wrong props type expectation, it should not be that opinionated I think, we should create an issue there